src/Neo.VM/Unsafe.cs

```diff
@@ -38,5 +40,40 @@ public static bool NotZero(ReadOnlySpan<byte> x)
             }
             return false;
         }
+
+        /// <summary>
+        /// Get 64-bit hash code for a byte array
+        /// </summary>
+        /// <param name="span">Span</param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static long HashBytes(ReadOnlySpan<byte> span)
+        {
+            var len = span.Length;
+            var hashState = (ulong)len;
+
+            fixed (byte* k = span)
+            {
+                var pwString = (char*)k;
+                var cbBuf = len / 2;
```

Comment:
@cschuchardt88 it doesn't iterate all the span? is that expected @cschuchardt88 ?

Reply:
yes